server/api/crud/model_monitoring/deployment.py

```diff
@@ -897,7 +897,7 @@ async def delete_monitoring_function(
                     function_names=[function_name],
                     access_key=access_key,
                 )
-            except mlrun.errors.MLRunStreamConnectionFailure as e:
+            except mlrun.errors.MLRunStreamConnectionFailureError as e:
                 logger.warning(
                     "Failed to delete stream resources, you may need to delete them manually",
                     project_name=project,
```

Comment:
Why adding error in the end (is it something in PEP saying that it should be done like this)? Cause it looks a bit redundant taking into account that it's already in `mlrun.errors` module..

Reply:
https://docs.astral.sh/ruff/rules/error-suffix-on-exception-name/
https://peps.python.org/pep-0008/#exception-names

In addition, please note almost all of the errors in `mlrun.errors` follow this guideline